cli/push.go

```diff
@@ -67,7 +65,7 @@ func (cli *DogestryCli) CmdPush(args ...string) error {
 
 // Stream the tarball from docker and translate it into the portable repo format
 // Note that its easier to handle as a stream on the way out.
-func (cli *DogestryCli) exportImageToFiles(image, root string) error {
+func (cli *DogestryCli) exportImageToFiles(image, root string, saveIds map[remote.ID]struct{}) error {
```

Comment:
We should typedef this as a set using `type set map[remote.ID]struct{}`? http://play.golang.org/p/SZhot5WUh4

It made sense after I read the comment down below, but it isn't quite as obvious just by looking at the code without the comment.